### PR TITLE
Support listing of employees in a given department

### PR DESCRIPTION
**Pull request by pkwangjoo**
- Support ListEmployeesInDepartmentCommand
- format: `leid n/DEPARTMENT_NAME`

### CODE REVIEW

File: src/test/java/seedu/sudohr/logic/commands/department/ListEmployeesInDepartmentCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    @Test
+    public void execute_nonExistingDepartmentThrows_throwsException() {
+        DepartmentName departmentName = DEPARTMENT_NAME_NOT_IN_SUDOHR;
+        ListEmployeesInDepartmentCommand command = new ListEmployeesInDepartmentCommand(departmentName);
+
+        assertCommandFailure(command, model, ListEmployeesInDepartmentCommand.MESSAGE_DEPARTMENT_NOT_FOUND);
+    }
+
+
```
Comment (kennethk-1201) [inline src/test/java/seedu/sudohr/logic/commands/department/ListEmployeesInDepartmentCommandTest.java:62]: Can you add a test case to execute the command on an empty department? thanks!

**Review by kennethk-1201 — APPROVED** [review]
Overall LGTM, made one comment on a possible test case, can just add that then ownself merge already
